test/bufr_ncep_satwind_avhrr.yaml

```diff
@@ -0,0 +1,186 @@
+#
+# SAID, CLAT, CLON, YEAR, MNTH, DAYS, HOUR, MINU, SECO, SWCM, SAZA, OGCE, SCCF, SWQM
+# HAMD, PRLC, WDIR, WSPD
+# GNAP, PCCF
+
+observations:
+  - obs space:
+      name: bufr
+      #obsdatain: "./testinput/bufr_read_2_dim_blocks.bufr"
+      obsdatain: "./testinput/bufr_ncep_satwind_avhrr.bufr_d"
+
+      exports:
+        variables:
+          satelliteIdentifier:
+            query: "*/SAID"
+
+          latitude:
+            query: "*/CLAT"
+
+          longitude:
+            query: "*/CLON"
+
+          year:
+            query: "*/YEAR"
+
+          month:
+            query: "*/MNTH"
+
+          day:
+            query: "*/DAYS"
+
+          hour:
+            query: "*/HOUR"
+
+          minute:
+            query: "*/MINU"
+
+          second:
+            query: "*/SECO"
+
+          satelliteDerivedWindComputationMethod:
+            query: "*/SWCM"
+
+          satelliteZenithAngle:
+            query: "*/SAZA"
+
+          originatingCenter:
+            query: "*/OGCE"
+
+          satelliteChannelCentreFrequency:
+            query: "*/SCCF"
+
+          sdmeditSatWindQualMark:
+            query: "*/SWQM"
+
+          heightAssignmentMethod:
+            query: "*/HAMD[1]"
+
+          pressure:
+            query: "*/PRLC[1]"
+
+          windDirection:
+            query: "*/WDIR"
+
+          windSpeed:
+            query: "*/WSPD"
+
+          generatingApplication:
+            query: "*/GQCPRMS/GNAP[1]"
+
+          percentConfidence:
+            query: "*/GQCPRMS/PCCF[1]"
+
+    ioda:
+      backend: netcdf
+      obsdataout: "./testoutput/bufr_ncep_satwind_avhrr.nc"
+      dimensions:
+        - name: "ncols_conf"
+          paths:
+            - "*/QCPRMS"
+
+      variables:
+        - name: "MetaData/satelliteIdentifier"
+          source: variables/satelliteIdentifier
+          longName: "Satellite Identifier"
+          units: ""
+
+        - name: "MetaData/latitude"
+          source: variables/latitude
+          longName: "Latitude (Coarse Accuracy)"
+          units: "degree_north"
+
+        - name: "MetaData/longitude"
+          source: variables/longitude
+          longName: "Longitude (Coarse Accuracy)"
+          units: "degree_east"
+
+        - name: "MetaData/year"
+          source: variables/year
+          longName: "Year"
+          units: "year"
+
+        - name: "MetaData/month"
+          source: variables/month
+          longName: "Month"
+          units: "mon"
+
+        - name: "MetaData/day"
+          source: variables/day
+          longName: "Day"
+          units: "day"
+
+        - name: "MetaData/hour"
+          source: variables/hour
+          longName: "Hour"
+          units: "hr"
+
+        - name: "MetaData/minute"
+          source: variables/minute
+          longName: "Minutes"
+          units: "min"
+
+        - name: "MetaData/second"
+          source: variables/second
+          longName: "Seconds"
+          units: "sec"
+
```

Comment:
Remove these lines and add the following:

```
        - name: "MetaData/dateTime"
          coordinates: "longitude latitude"
          source: variables/timestamp
          longName: "Datetime"
          units: "seconds since 1970-01-01T00:00:00Z"
```

Reply:
The mnemonics YEAR, MNTH, DAYS, HOUR and MINU from BUFR are used to construct EPOCH time (dateTime) in IODA.  The definition is seconds since 1970-01-01T00:00:00Z.

Reply:
@rmclaren @ilianagenkova Let's be a bit more careful in reviewing the PRs related to feature/query_cxx.  There is no hurry.  We have time .  Accuracy is more important than fast and more!

Reply:
Agree! I'll let @AshleyStanfield-NOAA address the PR suggested changes, and I'll clean up my old lines of code.

Reply:
The yaml has now been changed to match the above. Will be uploaded shortly.

Reply:
Sorry Emily. I just didn't see it.